client/src/components/MapView.vue

```diff
@@ -1,25 +1,63 @@
 <template>
   <div id="map-container"></div>
+  <MapLegend
+      :title="this.legendTitle"
+      :bucketMap="this.bucketMap"/>
 </template>
 
 <script>
 import mapbox from 'mapbox-gl';
 import axios from 'axios'
+import MapLegend from '@/components/MapLegend';
+import MapPopupContent from '@/components/MapPopupContent';
+import {createApp, defineComponent, nextTick} from 'vue'
 
 const DEFAULT_LATITUDE = 39.8097343;
 const DEFAULT_LONGITUDE = -98.5556199;
 
 const OPEN_DATA_PLATFORM_API_URL = 'https://v2rz6wzmb7.execute-api.us-east-2.amazonaws.com/default';
+const POPUP_CONTENT_BASE_ID = 'popup-content';
+const POPUP_CONTENT_BASE_HTML = `<div id="${POPUP_CONTENT_BASE_ID}"></div>`;
+
+// TODO(kaila): figure out useful buckets.
```

Comment:
These are probably too granular, but that's fine for now.

Reply:
Yeah I think for a final product it would be helpful to do some basic queries on the data to get an idea of the range of values that we should display here.